src/coreclr/tests/src/Interop/ArrayMarshalling/BoolArray/MarshalBoolArrayTest.csproj

```diff
@@ -8,6 +8,6 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="$(TestSourceDir)Common/CoreCLRTestLibrary/CoreCLRTestLibrary.csproj" />
-    <ProjectReference Include="CMakeLists.txt" />
+    <NativeProjectReference Include="MarshalBoolArrayNative/CMakeLists.txt" CMakeProject="MarshalBoolArrayNative/MarshalBoolArrayNative.proj" BuildNative="true" />
```

Comment:
had to move this to separate directory because we don't want more than one `proj` file in the same directory